app/test/unit/components/PasswordStrength/index.test.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import sinon from 'sinon'
+import * as zxcvbn from '$utils/zxcvbn'
+
+import PasswordStrength from '$shared/components/PasswordStrength'
+
+describe(PasswordStrength.name, () => {
+    const sandbox = sinon.createSandbox()
+
+    const getStrength = async (props) => (
+        mount(<PasswordStrength {...props} />).instance().getStrength()
+    )
+
+    const stubZxcvbn = (score) => {
+        sandbox.stub(zxcvbn, 'default').callsFake(() => Promise.resolve(() => ({
```

Comment:
why not let it actually use the module? e.g. if the `zxcvbn` module ever changes and breaks our code, this test wouldn't catch that. This page of tests are kinda just testing that this array is in this shape: `[0, 1, 1, 2, 2]`, which is just configuration details. i.e. I don't think tests should be breaking on configuration changes. IMO find some examples that meet these strength criteria. If the module API changes or it starts considering weak passwords as strong, then these tests would catch that. That seems valuable. Maybe?

Reply:
Yep, you're right. Will change that.